tests/README.md

```diff
@@ -26,3 +26,4 @@ python generate_test_cases.py
 The file `scripts/plier_util.R` was downloaded from
 [here](https://github.com/greenelab/multi-plier/blob/v0.2.0/util/plier_util.R).
 It is used to generate test data.
+
```

Comment:
Hm, I suspect this might be related to the package `rpy2`. All the necessary files were already generated with `generate_test_cases.py` so no need to run that again (the error you get is because you didn't download that file with `setup_data.py`).

Could you try to run that test file with python and see if it can load the necessary packages (`python tests/test_multiplier.py`)?